packages/backend/src/apps/formatter/index.ts

```diff
@@ -0,0 +1,17 @@
+import { IApp } from '@plumber/types'
+
+import actions from './actions'
+
+const app: IApp = {
+  name: 'Formatter',
+  key: 'formatter',
+  iconUrl: '{BASE_URL}/apps/formatter/assets/favicon.svg',
+  authDocUrl: 'https://guide.plumber.gov.sg/user-guides/actions/formatter',
+  baseUrl: '',
+  apiBaseUrl: '',
+  primaryColor: '000000',
+  actions,
+  description: `Format your data`,
```

Comment:
super nit to use single quotes cus just mild OCD
```suggestion
  description: 'Format your data',
```

Reply:
gamu xia